src/ring_theory/ideal/basic.lean

```diff
@@ -590,6 +590,12 @@ def lift (S : ideal α) (f : α →+* β) (H : ∀ (a : α), a ∈ S → f a = 0
 @[simp] lemma lift_mk (S : ideal α) (f : α →+* β) (H : ∀ (a : α), a ∈ S → f a = 0) :
   lift S f H (mk S a) = f a := rfl
 
+/-- Weaken the relation of a quotient.
+
+This is the `ideal.quotient` version of `quot.factor` -/
+def inclusion (S T : ideal α) (H : S ≤ T) : S.quotient →+* T.quotient :=
```

Comment:
I would call this `map` or `map_of_le` or something like that. `S.quotient -> T.quotient` is almost never injective, so calling it `inclusion` does not make sense.

Reply:
Also, you should add a simp lemma that says `inclusion S T h (S^.quotient.mk x) = T^.quotient.mk x`

Reply:
I renamed it as `factor` to keep it similar to `quot.factor`. I've added a simp lemma, it should appear in my next `git push`.

Reply:
I renamed it as `factor` to keep it similar to `quot.factor`. I've added the simp lemma `factor_mk`, it should appear in my next `git push`.